internal/srv/app.go

```diff
@@ -113,10 +114,13 @@ func attachRoleBinding(ctx context.Context, client *kubernetes.Clientset, namesp
 
 // newDeployment deploys a loadBalancer based upon the configuration provided
 // from the event that is processed.
-func (s *Server) newDeployment(lb *loadBalancer) error {
+func (s *Server) newDeployment(ctx context.Context, lb *loadBalancer) error {
+	ctx, span := otel.Tracer(instrumentationName).Start(ctx, "newDeployment")
```

Comment:
think there are some `attributes` we can add to the tracers to track. Such as `loadbalancer_id`?